src/sourmash/tax/tax_utils.py

```diff
@@ -24,19 +24,22 @@
            'aggregate_by_lineage_at_rank', 'format_for_krona',
            'write_krona', 'write_summary', 'write_classifications',
            'combine_sumgather_csvs_by_lineage', 'write_lineage_sample_frac',
-           'MultiLineageDB']
+           'MultiLineageDB', 'RankLineageInfo']
 
 from sourmash.logging import notify
 from sourmash.sourmash_args import load_pathlist_from_file
 
 # CTB: these could probably usefully be converted into dataclasses.
-QueryInfo = namedtuple("QueryInfo", "query_md5, query_filename, query_bp, query_hashes, total_weighted_hashes")
-SummarizedGatherResult = namedtuple("SummarizedGatherResult", "query_name, rank, fraction, lineage, query_md5, query_filename, f_weighted_at_rank, bp_match_at_rank, query_ani_at_rank, total_weighted_hashes")
-ClassificationResult = namedtuple("ClassificationResult", "query_name, status, rank, fraction, lineage, query_md5, query_filename, f_weighted_at_rank, bp_match_at_rank, query_ani_at_rank")
+QInfo = namedtuple("QInfo", "query_md5, query_filename, query_bp, query_hashes, total_weighted_hashes")
+SumGathInf = namedtuple("SumGathInf", "query_name, rank, fraction, lineage, query_md5, query_filename, f_weighted_at_rank, bp_match_at_rank, query_ani_at_rank, total_weighted_hashes")
```

Comment:
OK I gotta ask - what does `Inf` stand for?

Reply:
Info. I was just renaming the old tuples so I could use their names on the new classes without breaking everything :). These got deleted in the last PR

Reply:
ok, I think I got it!

I'm going to be annoying and ask for 20% more explanation in the header of the PRs - in this case, the history/backstory/motivation linked to PR numbers :). I know it can feel repetitive but pretend you're explaining the motivation for each and every PR to someone with no long-term memory and very litte attention span 😂

You can even copy paste it from one into the others!

Reply:
done - let me know if you want more details!

Reply:
thank you! this is great!